src/components/Chart/ChartSquare.js

```diff
@@ -3,7 +3,7 @@ import gradient from 'gradient-color';
 import React from 'react';
 import { getColor } from 'utils';
 
-import { COLOR_GRADIENT, CHART_SQUARE_TYPES } from '../../constants/constants';
+import { CHART_SQUARE_TYPES, COLOR_GRADIENT } from '../../constants/constants';
```

Comment:
I believe we can pickup this from 'constants' now with the new webpack alias resolving

Reply:
https://github.com/starter-code/binge/blob/master/webpack.config.js#L32